addons/crm_iap_lead_website/i18n/vi.po

```diff
@@ -150,7 +150,7 @@ msgstr "Nhập Regex hợp lệ."
 #. module: crm_iap_lead_website
 #: model:ir.model.fields,field_description:crm_iap_lead_website.field_crm_reveal_rule__contact_filter_type
 msgid "Filter On"
-msgstr "Filter On"
+msgstr "Bật lọc"
```

Comment:
@Toanduck123 Từ này truy cập chỗ nào có vậy em?

Reply:
![2021-11-08_13-57](https://user-images.githubusercontent.com/48975854/140697486-4e4d7f8f-8746-4551-ae71-af85273ef893.png)
Chị chọn `Companies and their Contacts` trên trường `data tracking` thì nó hiện ra phần này

Reply:
@Toanduck123 "Bật lọc" nghĩa là gì chú? Khi dịch đã xem ngữ cảnh của từ này chưa?

@jennie-pham ngữ cảnh của từ này như sau: trong module `crm_iap_lead_website (Lead Generation From Website Visits)` có model `crm.reveal.rule (CRM Lead Generation Rules)` nó dùng để thiết lập quy tắc generate lead, trong quá trình thiết lập thì nó cho phép mình cấu hình `contact_filter_type` trường này có 2 lựa chọn: `Role` hoặc `Seniority`. Và trường này khi hiển thị trên phần mềm nó hiển thị là `Filter On` để cho người dùng chọn 1 trong 2 lựa chọn trên. Do đó chỗ này có thể dịch là `Lọc Theo`

Reply:
vâng, em hiểu nhầm chỗ này ạ